client/src/pages/Profile.js

```diff
@@ -153,7 +148,7 @@ class Profile extends Component {
     render () {
         const { classes } = this.props;
```

Comment:
`const { classes, profile } = this.props;`
remove all this.props, for consistency?